File: PTIT127I.cpp
/*
* @Author: termi
* @Date:   2018-01-14 14:51:34
* @Last Modified by:   shentury
* @Last Modified time: 2018-01-14 15:32:57
*/
#include<bits/stdc++.h>
using namespace std;
int main()
{
	while(true)
	{
		int count=0;
		cin>>count;
		if(!count)
		{
			break;
		}
		else
		{
			int templat[51];
			for(int i=0;i<50;i++)
			{
				templat[i]=0;
			}
			for(int i=0;i<count;i++)
			{
				for(int j=0;j<6;j++)
				{
					long long  temp;
					cin>>temp;
					if(temp>=1&&temp<=49)
					{
						templat[temp]++;
					}
				}
			}
			bool flag=true;
			for(int i=1;i<=49;i++)
			{
				if(templat[i]==0)
				{
					flag=false;
				}
			}
			if(flag)
			{
				cout<<"Yes"<<endl;
			}
			else
			{
				cout<<"No"<<endl;
			}
		}
	}
}

File: PTIT127I.py
# -*- coding: utf-8 -*-
# @Author: termi
# @Date:   2018-01-14 14:29:41
# @Last Modified by:   termi
# @Last Modified time: 2018-01-14 14:47:51
while True :
	template=[]
	for index in range(51):
		template.append(0)
	count=int(raw_input())
	if count!=0:
		for index in range(count):
			for indexx in range(6):
				template[int(raw_input())]+=1
		flag=True
		for index in range(1,51):
			if template[index]==0:
				print 'No'
				flag=False
				break
		if flag :
			print 'Yes'
	else:
		break

	
